lib/dicom_net/association.ex

```diff
@@ -199,4 +232,64 @@ defmodule DicomNet.Association do
 
     response_ds
   end
+
+  defp cfind_response_header(command, presentation_context_id, status) do
+    asci_de = DataSet.fetch!(command, :AffectedSOPClassUID)
+    mid_de = DataSet.fetch!(command, :MessageID)
+
+    response =
+      Dicom.DataSet.from_keyword_list(
+        AffectedSOPClassUID: DataElement.value(asci_de),
+        CommandField: 0x8020,
+        MessageIDBeingRespondedTo: DataElement.value(mid_de),
+        CommandDataSetType: 0x0001, # This field shall be set to the value of 0101H (Null) if no Data Set is present; any other value indicates a Data Set is included in the Message.
+        Status: status # 0x0000 Success, 0xff00 Pending
+      )
+
+    Dicom.BinaryFormat.serialize_command_data_set(response)
+      |> Pdu.new_data_pdu(presentation_context_id) 
+      |> Pdu.serialize()
+  end
+
+  defp cfind_response_data(presentation_context_id, response) do
+    Dicom.BinaryFormat.serialize(response, [endianness: :little, explicit: true])
```

Comment:
Would add a `# TODO: options are hardcoded and need to be obtained from PC` comment here.